packages/mco/components/disaster-recovery/drpolicy-list-page/drpolicy-list-page.tsx

```diff
@@ -226,53 +135,64 @@ const DRPolicyList: React.FC<DRPolicyListProps> = (props) => {
   );
 };
 
-type DRPolicyListProps = {
-  data: DRPolicyKind[];
-  unfilteredData: DRPolicyKind[];
-  loaded: boolean;
-  loadError: any;
-  rowData?: any;
-};
-
 export const DRPolicyListPage: React.FC = () => {
   const { t } = useCustomTranslation();
-  const history = useHistory();
+  // UI will list drPolicies when user has read permission only for DRPolicy resource
+  const drResources = useDisasterRecoveryResourceWatch();
+  const [formattedDRResource] = useDisasterRecoveryFormatter({
+    ...drResources,
+    loadPartial: true,
+  });
```

Comment:
do not destructure, It will send new object to "useDisasterRecoveryFormatter" everytime component will re render...
```suggestion
  const [formattedDRResource] = useDisasterRecoveryFormatter(drResources, true);
```

Reply:
got it